bridges/snowbridge/pallets/system-v2/src/lib.rs

```diff
@@ -228,12 +228,10 @@ pub mod pallet {
 		fn send(origin: H256, command: Command, fee: u128) -> DispatchResult {
 			let mut message = Message {
 				origin,
-				id: Default::default(),
+				id: frame_system::unique((origin, command, fee)),
```

Comment:
I'd suggest using `blake2_256` for determinism.

Reply:
Addressed in https://github.com/claravanstaden/polkadot-sdk/pull/21/commits/78b732620e93d12e171f10f23a33e12fce4e23c2.

Reply:
I don't think determinism is useful in this case. These are governance messages emitted by BridgeHub itself. I don't think any UIs are going to try and deterministically recreate these message ids.

Reply:
Reverted https://github.com/claravanstaden/polkadot-sdk/commit/78b732620e93d12e171f10f23a33e12fce4e23c2.